### PR TITLE
Implementing GCN Class

### LINKED ISSUE #7: GCN : Create class for GCN
Define a class for the GCN and define its architecture:
- use 2-3 layer network using normalized adjacency

### PR DESCRIPTION
**Pull request by BrandonN24**
Resolves #7 
Features:
- Created networks/GCN.py file to define a two-layer and three-layer GCN
- adjusted root directory of dataset download for citeseer in training/citeseer.py

Layer features:
- ReLU and dropout at each layer
- Log Softmax output activation

### CODE REVIEW

**Review by simonbruce85 — APPROVED** [review]
lgtm
